### PR TITLE
Fix the errors in the stream service

### PR DESCRIPTION
**Pull request by solittlework**

### CODE REVIEW

File: apps/tenant-management-webapp/src/app/components/RoleTable.tsx [hunk 1]
```diff
@@ -48,8 +49,9 @@ export const ClientRoleTable = (props: ClientRoleTableProps): JSX.Element => {
   };
 
   const getClientId = () => {
-    return props.clientId ? <PaddingRem>{props.clientId}</PaddingRem> : tenantName;
+    return props.clientId && props.clientId !== REALM_ROLE_KEY ? props.clientId : tenantName;
```
Comment (jonathanweyermann) [inline apps/tenant-management-webapp/src/app/components/RoleTable.tsx:52]: can you put <PaddingRem> around props.clientId

File: apps/tenant-management-webapp/src/app/components/styled-components.ts [hunk 2]
```diff
@@ -2,11 +2,7 @@ import styled from 'styled-components';
 
 export const MarginAdjustment = styled.h4`
   margin-bottom: 0.5rem !important;
-  margin-top: 2rem !important;
-`;
-
-export const PaddingRem = styled.div`
-  padding-top: 1rem;
+  margin-top: 3rem !important;
 `;
```
Comment (jonathanweyermann) [inline apps/tenant-management-webapp/src/app/components/styled-components.ts:6]: don't do this change

**Review by jonathanweyermann — CHANGES_REQUESTED** [review]
couple comments

File: apps/tenant-management-webapp/src/app/components/styled-components.ts (hunk 2 above)
Comment (solittlework) [reply apps/tenant-management-webapp/src/app/components/styled-components.ts]: Sure, I will roll back it.

**Review by jonathanweyermann — APPROVED** [review]

**Pull request merged**